lms/templates/courseware/progress.html

```diff
@@ -97,6 +98,9 @@ <h2 class="title">${_("Congratulations, you qualified for a certificate!")}</h2>
 
       %if not course.disable_progress_graph:
         <div class="grade-detail-graph" id="grade-detail-graph" aria-hidden="true"></div>
+        %if not course.no_grade and any(grader_conf.get('passing_grade') and grader_conf.get('passing_grade_enabled') for grader_conf in course.raw_grader):
```

Comment:
what the reason to check for `grader_conf.get('passing_grade')`? I though `grader_conf.get('passing_grade_enabled')` have to be enough.